bedwars-api/src/main/java/com/andrei1058/bedwars/api/server/VersionSupport.java

```diff
@@ -435,4 +435,6 @@ public void spigotHidePlayer(Player victim, Player receiver) {
     public abstract void playRedStoneDot(Player player);
 
     public abstract void clearArrowsFromPlayerBody(Player player);
+
+    public abstract void playEffect(Player player, Location location);
```

Comment:
This must be called playVillagerEffect or something like this

Reply:
Fixed